### DISCUSSION TITLE
Remember to review and, if possible, merge your classmates PRs

### DISCUSSION THREAD

**Discussion by metasean** [USU-ITLS-x265-23-summer/class-repo#51] [Announcements]
What the title says 😉

**Comment by bst3ck** [comment]
Thanks for the note. I had thought the original submitter was responsible to merge their own pull requests and not the approver. I will make sure to do that going forward.

**Reply by metasean** [reply]
Technically, it can be done either way.  

For this class, because this repo should be getting a lot of commits that are in the same areas of code, having the reviewer make the merge should reduce the likelihood of different PRs conflicting with each other.
